packages/spawner/src/contractHost.js

```diff
@@ -10,7 +8,7 @@ import { makeIssuerKit } from '@agoric/ertp';
 
 export { makeCollect } from './makeCollect';
 
-/**
+/*
```

Comment:
Why remove the second star?

Reply:
This is not a valid jsdoc, since it contains commentary but doesn’t declare the signature. The file doesn’t have `@ts-check` so it’s okay for it to be informal, but in normalizing the lint rules, I entrained the `jsdoc` eslint rules, which require these blocks to be valid.